tests/ci/cpu_unit_tests.yml

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
are you planning to add the gpu and spark files in this PR or in a different one? We have 3 environments, 2 OS, 2 branches, and 3 kinds of tests (unit, notebooks and nightly). How are we going to setup all this?

Reply:
Different one. My plan is to get this PR in, then I've got others waiting behind this one for:
- template - to abstract common code for the other items you mention
- job - another useful pipeline abstraction I have ready to submit
- pipelines on other branch - I've got another branch where I set up the 3 envs and 3 kinds of tests. I can show you those.  Spark has some issues that I believe are setup issues. The others have a few test fails. I want to get this PR in and then those will follow. 
- About Windows: AzureML only runs linux.  We'll need to

Reply:
About Windows: AzureML only runs linux. We'll need to talk about what to do with Windows.